utils/docker-ce/Makefile

```diff
@@ -1,15 +1,15 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=docker-ce
-PKG_VERSION:=18.09.6
-PKG_RELEASE:=1
+PKG_VERSION:=18.09.7
+PKG_RELEASE:=2
```

Comment:
`PKG_RELEASE` should be kept at 1.  And when `PKG_VERSION` was bumped, `PKG_RELEASE` should be reset to 1.

Reply:
oops, missed that when I updated.
Reverted.